pkgs/by-name/sc/scooter/package.nix

```
@@ -0,0 +1,25 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "scooter";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "thomasschafer";
+    repo = "scooter";
+    rev = "v${version}";
+    hash = "sha256-5YosLJHfrXv6ev7r0gAW5bzkSIwIBUj/Fl3+tklTKQo=";
+  };
+
+  cargoHash = "sha256-GY52JN07igDJVuKXR9iIqNj1obhPs70dOmU1wGy8ZrI";
+
+  meta = with lib; {
+    description = "Interactive find and replace in the terminal";
+    homepage = "https://github.com/thomasschafer/scooter";
+    changelog = "https://github.com/thomasschafer/scooter/commits/v${version}";
+    license = with licenses; [ mit ];
+    maintainers = with maintainers; [ felixzieger ];
```

```suggestion
    maintainers = with lib.maintainers; [ felixzieger ];
```
See https://github.com/NixOS/nixpkgs/issues/208242 https://github.com/NixOS/nixpkgs/pull/293767